lib/numruby.rb

```diff
@@ -0,0 +1,5 @@
+require_relative 'numruby/numruby.rb'
+require_relative 'numruby/nmatrix.rb'
```

Comment:
I'm not sure, but I think that you also should update the name of this file. WDYT? 🤔

Reply:
`lib/numruby.rb` is for whole gem, hence denoting the gem name.
`lib/numruby/numruby.rb` is for NumRuby class.
`lib/numruby/nmatrix.rb` is for NMatrix class that represents the matrix object class.

It's analogous to NumPy and Ndarray.